### DISCUSSION TITLE
Runtime NetworkBehaviours

### DISCUSSION THREAD

**Discussion by FirstGearGames** [FirstGearGames/FishNet#418] [Feature Request]
Allow NetworkBehaviours to be added at runtime.

This has been tested to a degree and should be possible. At this time however this feature is a low priority.

**Comment by stilnat** [comment]
In my opinion, it'd be a great adding to fishnet, we at [SS3D](https://github.com/RE-SS3D) feel limited in some aspects of our development without the possibility to add NetworkBehaviours at runtime.
Just writing that here as maybe it'll make a small incentive to reconsider priority.

**Comment by FirstGearGames** [comment]
> In my opinion, it'd be a great adding to fishnet, we at [SS3D](https://github.com/RE-SS3D) feel limited in some aspects of our development without the possibility to add NetworkBehaviours at runtime. Just writing that here as maybe it'll make a small incentive to reconsider priority.

Very neat project! New features are mostly off the table until prediction v2 is out of experimental.

**Reply by Katerlad** [reply]
Have thought of a few areas that I was going to do this, and then realized I couldn't. +1 from me

**Comment by FirstGearGames** [comment]
Added to roadmap.
